### PR TITLE
Prepare readme update

### PR DESCRIPTION
**Pull request by davisshaver**
See #6

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -71,12 +71,23 @@ Publishing_Checklist::get_instance();
...
+= Where will the checklist appear? =
+
+On Manage and Edit post screens.
+
 = Does the plugin come with any default checklists? =
 
 Not yet.
 
+
 == Screenshots ==
```
Comment (danielbachhuber) [inline readme.txt]: Screenshots need to be named `screenshot-1.png`, `screenshot-2.png` and live in the root directory. GIFs aren't supported.

**Comment by davisshaver** [comment]
@daniel it wasn't clear from the sample that the `screenshot-1.png` convention is required. Also the sample indicates that `.gif` is valid.

**Comment by davisshaver** [comment]
https://wordpress.org/plugins/about/readme.txt

**Comment by davisshaver** [comment]
@daniel Based on the readme.txt sample, I think the image files are correctly placed in `assets`...

**Comment by danielbachhuber** [comment]
Nope, you can trust me on this one.

**Comment by danielbachhuber** [comment]
The /assets directory mentioned correlates with the SVN repo, where this repo correlates with /trunk in the SVN repo

**Comment by davisshaver** [comment]
@danielbachhuber review?

**Comment by danielbachhuber** [comment]
wth

**Comment by davisshaver** [comment]
@danielbachhuber weird... I fixed the PHPCS issue, should be good to merge. Not sure why it wasn't caught earlier.

**Comment by danielbachhuber** [comment]
Thanks @davisshaver

**Pull request merged**